blockstack-sdk/src/main/res/layout/activity_connect.xml

```diff
@@ -0,0 +1,186 @@
+                android:src="@drawable/ic_lock"
+                android:tint="?colorPrimary"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="@id/connect_secret_key_desc_title" />
+
+            <TextView
+                android:id="@+id/connect_secret_key_desc_title"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="16dp"
+                android:layout_marginRight="16dp"
+                android:gravity="top"
+                android:text="@string/connect_activity_encryption_desc"
+                android:textAppearance="@style/TextAppearance.AppCompat.Medium"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toEndOf="@id/connect_secret_key_desc_icon"
+                app:layout_constraintTop_toBottomOf="@id/secret_key_divider" />
+
+            <View
+                android:id="@+id/tracking_divider"
+                style="?blockstackDivider"
+                android:layout_marginTop="16dp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/connect_secret_key_desc_title" />
+
+            <ImageView
+                android:id="@+id/connect_tracking_desc_icon"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="4dp"
+                android:importantForAccessibility="no"
+                android:paddingLeft="16dp"
+                android:paddingRight="16dp"
+                android:src="@drawable/ic_track"
+                android:tint="?colorPrimary"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="@id/connect_tracking_desc_title" />
+
+            <TextView
+                android:id="@+id/connect_tracking_desc_title"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="16dp"
+                android:layout_marginEnd="16dp"
+                android:gravity="top"
+                android:text="@string/connect_activity_tracking_desc"
+                android:textAppearance="@style/TextAppearance.AppCompat.Medium"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toEndOf="@+id/connect_tracking_desc_icon"
+                app:layout_constraintTop_toBottomOf="@+id/tracking_divider" />
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+    </androidx.core.widget.NestedScrollView>
+
+
+    <com.google.android.material.button.MaterialButton
+        style="?blockstackMainButtonStyle"
+        android:id="@+id/connect_get_secret_key"
+        android:layout_height="50dp"
```

Comment:
50dp does not fit the 8dp grid.

Reply:
asked by claudia to be the exact same as their popup. but I will do a multiple